aspnetcore/mvc/razor-pages/index.md

```diff
@@ -256,9 +256,9 @@ Add a *_ViewImports.cshtml* file:
 @addTagHelper "*, Microsoft.AspNetCore.Mvc.TagHelpers"
 ```
 
-The `@namespace` directive is a new feature that controls the namespace of the generated code allowing us to get rid of `@using` directives. If the `@namspace` directive is used directly in the page, the page will be put directly in the specified namespace. If the `@namespace` directive is inside a *_ViewImports.cshtml*, the specified namespace is only the prefix. The suffix is the dot-seperated relative path between the folder containing *_ViewImports.cshtml* and the folder containing the page.
+The `@namespace` directive is a new feature that controls the namespace of the generated code.  The `@namespace` directive replaces  `@using` directives. If the `@namespace` directive is used explicitly on a page, the page is in scope for the specified namespace. If the `@namespace` directive is contained in *_ViewImports.cshtml*, the specified namespace is only the prefix. The suffix is the dot-seperated relative path between the folder containing *_ViewImports.cshtml* and the folder containing the page.
```

Comment:
How about the following wording? "The `@namespace` directive allows for removal of some `@using` directives." I wouldn't even mind removing the sentence entirely, since most C# developers should understand this about namespaces.

I also like changing the "in scope for" message to "If the `@namespace` directive is used explicitly on a page, that page's namespace will match specified namespace exactly."